eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -45,6 +45,9 @@ extends:
       policy: M365
       credscan:
         toolVersion: 2.3.12.23
+      sourceRepositoriesToScan:
+        exclude:
+          - repository: SwiftPM-AzureTemplate
```

Comment:
Why did you exclude this?

Reply:
To avoid compilation errors when having an additional checkout.

Reply:
I'll merge this now and see if I can reverse this to explicitly include it.